scripts/rook.sh

```diff
@@ -26,7 +26,15 @@ function deploy_rook() {
 		kubectl create -f "${TEMP_DIR}/cluster-test.yaml"
 		rm -rf "${TEMP_DIR}"
 	else
-		kubectl create -f "${ROOK_URL}/cluster-test.yaml"
+		# add "mon_warn_on_pool_no_redundancy = false" to ceph.conf if missing
+		# see https://github.com/rook/rook/pull/5925 for upstream status
+		TEMP_DIR="$(mktemp -d)"
+		curl -o "${TEMP_DIR}"/cluster-test.yaml "${ROOK_URL}/cluster-test.yaml"
+		if ! grep -q mon_warn_on_pool_no_redundancy "${TEMP_DIR}"/cluster-test.yaml; then
```

Comment:
@nixpanic `mon_warn_on_pool_no_redundancy` can be declared as a const?

Reply:
how do you mean? this is an option for ceph.conf.

Reply:
I understand its an option.. What I meant was using something like `ROOK_CEPH_CLUSTER_VERSION_IMAGE_PATH` and using it in `sed` command.

Reply:
I do not think there is any benefit from that. The value is not configurable or constructed dynamically. Replacing it with a variable makes the code more difficult to understand, imho. The value is used only twice, and they are close together, a typo is unlikely.

Reply:
The other advantage is that, it (top level declared var) helps some one to understand there is some where the changes happening or its getting referred  and will be mindful about it. If these strings hide in between you may get miss those as well. I am fine either way.